Documentation/project-docs/dogfooding.md

```diff
@@ -17,11 +17,38 @@ this experience. Make sure to consult this document often.
 To setup the SDK download the zip and extract it somewhere and add the root folder to your path or always fully
 qualify the path to dotnet in the root of this folder for all the instructions in this document.
```

Comment:
same here. This paragraphs intent is the same as below where you describe to take care when using a local copy of the cli to fully qualify the path to dotnet.